docs/user/device-groups.md

```diff
@@ -12,6 +12,12 @@ navTitle: Device Groups
 When managing many devices that are intended to run the same [snapshot](./snapshots.md), Device Groups allow you
 to organise your devices into logical groups.
 These groups can then be set as the target of a [DevOps Pipelines](./devops-pipelines.md).
+
+Furthermore:
+* Devices added to an active Device Group will automatically be updated to the active pipeline snapshot
+* Devices removed from an active Device Group will have their active pipeline snapshot cleared
+More details are provided below.
```

Comment:
Can we link these to the anchors? Or do h3 elements not get an anchor by default? "More details provided below" works in this diff as the examples are marked in a nice green shade, but I'm not sure the end-user knows where to scroll to.

Reply:
@ZJvandeWeg 

Updated in [769e49c](https://github.com/FlowFuse/flowfuse/pull/3330/commits/769e49c44463ff5926cd1946cd5338d7f0cce68b)

Please see how that reads now.